Routes/Routes.go

```diff
@@ -0,0 +1,131 @@
+package route
+
+//hellow world
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	models "main/Models"
+	sqlconnect "main/SqlConnection"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+var db *sql.DB
+
+// CreateStudent godoc
+// @Summary Create a new Student
+// @Description Create a new Student and stores it in the database
+// @Tags Studetn
+// @Accept  json
+// @Produce  json
+// @Param Studentinfo body StudentInfo true "Create Student"
+// @Success 200 {object} Studentinfo
+// @Router /students [post]
+func AddStudents(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
Create Constant package which contains constant.go.
Inside that file define constant for "Content-Type", "application/json" use those constants.